libs/gl-client/Cargo.toml

```diff
@@ -35,7 +35,7 @@ serde_json = "^1.0"
 thiserror = "1"
 cln-grpc = { git = "https://github.com/ElementsProject/lightning.git", branch = "master"}
 chacha20poly1305 = { version = "0.10.1", optional = true }
-futhark = { git = "https://github.com/nepet/futhark.git", branch = "main"}
+futhark = { git = "https://github.com/nepet/futhark.git", commit = "611bcab"}
```

Comment:
Is this on purpose? Pointing at a branch sounds safer since that can move and sync up with the branch, whereas pointing at a commit may cause confusion when developing on the remote repo, but changes don't land in the client.

Reply:
Good catch! This was to avoid caching issues, I will change it back to master :+1: